### PR TITLE
Danse's Truly Amazing Gun Changes

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
WIP. There's still some things that need to be sorted out and there are some balance considerations to think about, for example the amount of magazines security gets for their .45s (now .40s) may need to be changed, and I may need to mess around with the snowflake guns we have that use 10mm. 
All done in accordance with: https://docs.google.com/document/d/18-mDVbO1DGXNjOGBAJH2GSe92LjMbKkY-d6SDox7ENU/edit

Also includes a huge nerf to full auto fire! Rejoice.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!wip

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/projectile/automatic
 	name = "prototype SMG"
-	desc = "A protoype lightweight, fast firing gun. Uses 9mm rounds."
+	desc = "A protoype lightweight, fast firing gun. Uses .35 "Light" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:3]:
```suggestion
	desc = "A protoype lightweight, fast firing gun. Uses .35\ "Light\" rounds."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 2]
```diff
@@ -26,8 +26,8 @@
 //Submachine guns and personal defence weapons, go.
 
 /obj/item/gun/projectile/automatic/mini_uzi
-	name = ".45 machine pistol"
-	desc = "A lightweight, fast firing gun. For when you want someone dead. Uses .45 rounds."
+	name = ".40 machine pistol"
+	desc = "A lightweight, fast firing gun. For when you want someone dead. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:30]:
```suggestion
	desc = "A lightweight, fast firing gun. For when you want someone dead. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 3]
```diff
@@ -46,7 +46,7 @@
 
 /obj/item/gun/projectile/automatic/c20r
 	name = "submachine gun"
-	desc = "The C-20r is a lightweight and rapid firing SMG, for when you REALLY need someone dead. Uses 10mm rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
+	desc = "The C-20r is a lightweight and rapid firing SMG, for when you REALLY need someone dead. Uses .40 "Standard" rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:49]:
```suggestion
	desc = "The C-20r is a lightweight and rapid firing SMG, for when you REALLY need someone dead. Uses .40 \"Standard\" rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 4]
```diff
@@ -72,7 +72,7 @@
 
 /obj/item/gun/projectile/automatic/wt550
 	name = "machine pistol"
-	desc = "The NI 550 Saber is a cheap self-defense weapon, mass-produced by Zavodskoi Interstellar for paramilitary and private use. Uses 9mm rounds."
+	desc = "The NI 550 Saber is a cheap self-defense weapon, mass-produced by Zavodskoi Interstellar for paramilitary and private use. Uses .35 "Light" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:75]:
```suggestion
	desc = "The NI 550 Saber is a cheap self-defense weapon, mass-produced by Zavodskoi Interstellar for paramilitary and private use. Uses .35 \"Light\" rounds."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 5]
```diff
@@ -134,8 +134,8 @@
 
 /obj/item/gun/projectile/automatic/rifle/sts35
 	name = "assault rifle"
-	desc = "A durable, rugged looking automatic weapon of a make popular on the frontier worlds. Uses 7.62mm rounds. It is unmarked."
-	desc_fluff = "The STS35 is a durable, reliable and cheap to buy fully automatic assault rifle with many licensed manufacturers across the galaxy. It comes in different versions and calibres, this one uses 7.62 rounds. The manufacturer markings have been filed off."
+	desc = "A durable, rugged looking automatic weapon of a make popular on the frontier worlds. Uses .315 "Rifle" rounds. It is unmarked."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:137]:
```suggestion
	desc = "A durable, rugged looking automatic weapon of a make popular on the frontier worlds. Uses .315 \"Rifle\" rounds. It is unmarked."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 6]
```diff
@@ -134,8 +134,8 @@
...
+	desc_fluff = "The STS35 is a durable, reliable and cheap to buy fully automatic battle rifle with many licensed manufacturers across the galaxy. It comes in different versions and calibres, this one uses .315 "Rifle" rounds. The manufacturer markings have been filed off."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:138]:
```suggestion
	desc_fluff = "The STS35 is a durable, reliable and cheap to buy fully automatic battle rifle with many licensed manufacturers across the galaxy. It comes in different versions and calibres, this one uses .315 \"Rifle\" rounds. The manufacturer markings have been filed off."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 7]
```diff
@@ -151,7 +151,7 @@
 
 /obj/item/gun/projectile/automatic/rifle/sol
 	name = "battle rifle"
-	desc = "A powerful battle rifle, the M469 is a highly accurate skirmishing firearm of Zavodskoi Instellar make which is chambered in 7.62."
+	desc = "A powerful battle rifle, the M469 is a highly accurate and highly outdated skirmishing firearm of Zavodskoi Instellar make which is chambered in .315 "Rifle" calibre."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:154]:
```suggestion
	desc = "A powerful battle rifle, the M469 is a highly accurate and highly outdated skirmishing firearm of Zavodskoi Instellar make which is chambered in .315 \"Rifle\" calibre."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 8]
```diff
@@ -179,7 +179,7 @@
 
 /obj/item/gun/projectile/automatic/rifle/z8
 	name = "bullpup assault carbine"
-	desc = "The Z8 Bulldog bullpup carbine, made by the now defunct Zendai Foundries. Uses armor piercing 5.56mm rounds. Makes you feel like a space marine when you hold it."
+	desc = "The Z8 Bulldog bullpup carbine, made by the now defunct Zendai Foundries. Uses .250 "Intermediate" rounds. Makes you feel like a space marine when you hold it."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:182]:
```suggestion
	desc = "The Z8 Bulldog bullpup carbine, made by the now defunct Zendai Foundries. Uses .250 \"Intermediate\" rounds. Makes you feel like a space marine when you hold it."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 9]
```diff
@@ -365,7 +365,7 @@
 
 /obj/item/gun/projectile/automatic/tommygun
 	name = "vintage submachine gun"
-	desc = "A classic submachine gun. Uses .45 rounds."
+	desc = "A classic submachine gun, copied by the Tajara. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/automatic.dm:368]:
```suggestion
	desc = "A classic submachine gun, copied by the Tajara. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/improvised.dm [hunk 10]
```diff
@@ -137,7 +137,7 @@
 
 /obj/item/gun/projectile/improvised_handgun
 	name = "improvised handgun"
-	desc = "A common sight in an amateur's workshop, a simple yet effective assembly made to chamber and fire .45 Rounds."
+	desc = "A common sight in an amateur's workshop, a simple yet effective assembly made to chamber and fire .40 "Standard" Rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/improvised.dm:140]:
```suggestion
	desc = "A common sight in an amateur's workshop, a simple yet effective assembly made to chamber and fire .40 \"Standard\" Rounds."
```

File: code/modules/projectiles/guns/projectile/improvised.dm [hunk 11]
```diff
@@ -207,7 +207,7 @@
 
 /obj/item/gun/projectile/automatic/improvised
 	name = "improvised machine pistol"
-	desc = "An improvised automatic handgun. Uses .45 rounds."
+	desc = "An improvised automatic handgun. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/improvised.dm:210]:
```suggestion
	desc = "An improvised automatic handgun. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 12]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/projectile/colt
-	name = "vintage .45 pistol"
-	desc = "A cheap Martian knock-off of a Colt M1911. Uses .45 rounds."
+	name = "vintage .40 pistol"
+	desc = "A cheap Martian knock-off of a truly ancient gun. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:3]:
```suggestion
	desc = "A cheap Martian knock-off of a truly ancient gun. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 13]
```diff
@@ -41,9 +41,9 @@
 		return 1
 
 /obj/item/gun/projectile/sec
-	name = ".45 pistol"
-	desc = "A NanoTrasen designed sidearm, found among law enforcement and security forces. Uses .45 rounds."
-	desc_fluff = "The NT Mk58 is a ballistic sidearm developed and produced by Nanotrasen. Bulky and heavy, the Mk58 is nonetheless used by security forces and law enforcement for its ease of use, low maintenance requirement, longevity, reliability - and most of all, extremely inexpensive price tag. A trademark of Nanotrasen security forces. It uses .45 rounds."
+	name = ".40 pistol"
+	desc = "A NanoTrasen designed sidearm, found among law enforcement and security forces. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:45]:
```suggestion
	desc = "A NanoTrasen designed sidearm, found among law enforcement and security forces. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 14]
```diff
@@ -41,9 +41,9 @@
...
+	desc_fluff = "The NT Mk58 is a ballistic sidearm developed and produced by Nanotrasen. Bulky and heavy, the Mk58 is nonetheless used by security forces and law enforcement for its ease of use, low maintenance requirement, longevity, reliability - and most of all, extremely inexpensive price tag. A trademark of Nanotrasen security forces. It uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:46]:
```suggestion
	desc_fluff = "The NT Mk58 is a ballistic sidearm developed and produced by Nanotrasen. Bulky and heavy, the Mk58 is nonetheless used by security forces and law enforcement for its ease of use, low maintenance requirement, longevity, reliability - and most of all, extremely inexpensive price tag. A trademark of Nanotrasen security forces. It uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 15]
```diff
@@ -66,13 +66,13 @@
...
 /obj/item/gun/projectile/sec/flash
-	name = ".45 signal pistol"
+	name = ".40 signal pistol"
 	magazine_type = /obj/item/ammo_magazine/c45m/flash
 
 /obj/item/gun/projectile/sec/wood
-	desc = "A NanoTrasen designed sidearm, found among law enforcement and security forces. It has a wooden grip. Uses .45 rounds."
-	desc_fluff = "The NT Mk58 is a ballistic sidearm developed and produced by Nanotrasen. Bulky and heavy, the Mk58 is nonetheless used by security forces and law enforcement for its ease of use, low maintenance requirement, longevity, reliability - and most of all, extremely inexpensive price tag. A trademark of Nanotrasen security forces. This one has a faux wooden grip. It uses .45 rounds."
-	name = "custom .45 Pistol"
+	desc = "A NanoTrasen designed sidearm, found among law enforcement and security forces. It has a wooden grip. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:73]:
```suggestion
	desc = "A NanoTrasen designed sidearm, found among law enforcement and security forces. It has a wooden grip. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 16]
```diff
@@ -66,13 +66,13 @@
...
+	desc_fluff = "The NT Mk58 is a ballistic sidearm developed and produced by Nanotrasen. Bulky and heavy, the Mk58 is nonetheless used by security forces and law enforcement for its ease of use, low maintenance requirement, longevity, reliability - and most of all, extremely inexpensive price tag. A trademark of Nanotrasen security forces. This one has a faux wooden grip. It uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:74]:
```suggestion
	desc_fluff = "The NT Mk58 is a ballistic sidearm developed and produced by Nanotrasen. Bulky and heavy, the Mk58 is nonetheless used by security forces and law enforcement for its ease of use, low maintenance requirement, longevity, reliability - and most of all, extremely inexpensive price tag. A trademark of Nanotrasen security forces. This one has a faux wooden grip. It uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 17]
```diff
@@ -85,8 +85,8 @@
 		icon_state = "secgunwood-e"
 
 /obj/item/gun/projectile/automatic/x9
-	name = "automatic .45 pistol"
-	desc = "The x9 tactical pistol is a lightweight fast firing handgun. Uses .45 rounds."
+	name = "automatic .40 pistol"
+	desc = "The x9 tactical pistol is a lightweight fast firing handgun. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:89]:
```suggestion
	desc = "The x9 tactical pistol is a lightweight fast firing handgun. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 18]
```diff
@@ -105,14 +105,14 @@
 	auto_eject_sound = 'sound/weapons/smg_empty_alarm.ogg'
 
 /obj/item/gun/projectile/tanto
-	desc = "A Zavodskoi Interstellar Tanto .40, designed to compete with the NT Mk58. Uses 10mm rounds."
-	name = "10mm auto-pistol"
+	desc = "A Zavodskoi Interstellar Tanto .40, designed to compete with the NT Mk58. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:108]:
```suggestion
	desc = "A Zavodskoi Interstellar Tanto .40, designed to compete with the NT Mk58. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 19]
```diff
@@ -134,7 +134,7 @@
 
 /obj/item/gun/projectile/silenced
 	name = "silenced pistol"
-	desc = "A small, quiet, easily concealable gun. Uses .45 rounds."
+	desc = "A small, quiet, easily concealable gun. Uses .40 "Standard" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:137]:
```suggestion
	desc = "A small, quiet, easily concealable gun. Uses .40 \"Standard\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 20]
```diff
@@ -217,8 +217,8 @@
 		icon_state = "gyropistol"
 
 /obj/item/gun/projectile/pistol
-	name = "9mm pistol"
-	desc = "A compact 9mm handgun, extremely popular all throughout human space."
+	name = ".35 pistol"
+	desc = "A compact .35 "Light" handgun, extremely popular all throughout human space."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:221]:
```suggestion
	desc = "A compact .35 \"Light\" handgun, extremely popular all throughout human space."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 21]
```diff
@@ -235,12 +235,12 @@
...
 
 /obj/item/gun/projectile/pistol/flash
-	name = "9mm signal pistol"
+	name = ".35 signal pistol"
 	magazine_type = /obj/item/ammo_magazine/mc9mm/flash
 
 /obj/item/gun/projectile/pistol/detective
-	desc = "A compact NanoTrasen designed sidearm, popular with law enforcement personnel for concealed carry purposes. It has a faux wooden grip. Uses 9mm rounds."
-	desc_fluff = "The NT Mk21 Blackjack is a ballistic sidearm developed and produced by Nanotrasen. Unlike the related Mk58, the Blackjack is a rather high quality piece - typically issued to higher ranking law enforcement personnel, the Mk21 is compact and chambered in 9mm caliber. With all the bells and whistles of a modern, quality police pistol, the Blackjack's main drawback is the notoriously nippy recoil - 9mm in such a small package can be unpleasant for the average shooter."
+	desc = "A compact NanoTrasen designed sidearm, popular with law enforcement personnel for concealed carry purposes. It has a faux wooden grip. Uses .35 "Light" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:242]:
```suggestion
	desc = "A compact NanoTrasen designed sidearm, popular with law enforcement personnel for concealed carry purposes. It has a faux wooden grip. Uses .35 \"Light\" rounds."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 22]
```diff
@@ -235,12 +235,12 @@
...
+	desc_fluff = "The NT Mk21 Blackjack is a ballistic sidearm developed and produced by Nanotrasen. Unlike the related Mk58, the Blackjack is a rather high quality piece - typically issued to higher ranking law enforcement personnel, the Mk21 is compact and chambered in .35 caliber. With all the bells and whistles of a modern, quality police pistol, the Blackjack's main drawback is the notoriously nippy recoil - .35 "Light" in such a small package can be unpleasant for the average shooter."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:243]:
```suggestion
	desc_fluff = "The NT Mk21 Blackjack is a ballistic sidearm developed and produced by Nanotrasen. Unlike the related Mk58, the Blackjack is a rather high quality piece - typically issued to higher ranking law enforcement personnel, the Mk21 is compact and chambered in .35 caliber. With all the bells and whistles of a modern, quality police pistol, the Blackjack's main drawback is the notoriously nippy recoil - .35 \"Light\" in such a small package can be unpleasant for the average shooter."
```

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 23]
```diff
@@ -348,12 +348,12 @@
 	. = ..()
 
 /obj/item/gun/projectile/leyon
-	name = "10mm pistol"
-	desc = "The Leyon LCC Everyman is a small pistol that holds five shots and is loaded with a stripper clip, popular for self-defense on Mars. Uses 10mm rounds."
+	name = "holdout .35 pistol"
+	desc = "The Leyon LCC Everyman is a small pistol that holds five shots and is loaded with a stripper clip, popular for self-defense on Mars. Uses .35 "Light" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm:352]:
```suggestion
	desc = "The Leyon LCC Everyman is a small pistol that holds five shots and is loaded with a stripper clip, popular for self-defense on Mars. Uses .35 \"Light\" rounds."
```

File: code/modules/projectiles/guns/projectile/rifle.dm [hunk 24]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/projectile/shotgun/pump/rifle
 	name = "bolt action rifle"
-	desc = "A cheap ballistic rifle often found in the hands of crooks and frontiersmen. Uses 7.62mm rounds."
+	desc = "A cheap ballistic rifle often found in the hands of crooks and frontiersmen. Uses .315 "Rifle" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/rifle.dm:3]:
```suggestion
	desc = "A cheap ballistic rifle often found in the hands of crooks and frontiersmen. Uses .315 \"Rifle\" rounds."
```

File: code/modules/projectiles/guns/projectile/rifle.dm [hunk 25]
```diff
@@ -55,7 +55,7 @@
 
 /obj/item/gun/projectile/contender
 	name = "pocket rifle"
-	desc = "A perfect, pristine replica of an ancient one-shot hand-cannon. This one has been modified to work almost like a bolt-action. Uses 5.56mm rounds."
+	desc = "A perfect, pristine replica of an ancient one-shot hand-cannon. This one has been modified to work almost like a bolt-action. Uses .250 "Intermediate" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/rifle.dm:58]:
```suggestion
	desc = "A perfect, pristine replica of an ancient one-shot hand-cannon. This one has been modified to work almost like a bolt-action. Uses .250 \"Intermediate\" rounds."
```

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 26]
```diff
@@ -180,7 +180,7 @@
 
 /obj/item/gun/projectile/dragunov
 	name = "antique sniper rifle"
-	desc = "An old semi-automatic marksman rifle. Uses 7.62mm rounds."
+	desc = "An old semi-automatic marksman rifle. Uses .315 "Rifle" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/sniper.dm:183]:
```suggestion
	desc = "An old semi-automatic marksman rifle. Uses .315 \"Rifle\" rounds."
```

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 27]
```diff
@@ -237,7 +237,7 @@
 
 /obj/item/gun/projectile/automatic/rifle/w556
 	name = "scout rifle"
-	desc = "A lightweight Neyland 556mi 'Ranger' used within the Sol Navy and Nanotrasen Emergency Response Teams. Equipped with a scope and designed for medium to long range combat, with moderate stopping power. Chambered in 5.56 rounds."
+	desc = "A lightweight Neyland 'Ranger' used within the Sol Navy and Nanotrasen Emergency Response Teams. Equipped with a scope and designed for medium to long range combat, with moderate stopping power. Chambered in .250 "Intermediate" rounds."
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/sniper.dm:240]:
```suggestion
	desc = "A lightweight Neyland 'Ranger' used within the Sol Navy and Nanotrasen Emergency Response Teams. Equipped with a scope and designed for medium to long range combat, with moderate stopping power. Chambered in .250 \"Intermediate\" rounds."
```

**Comment by TheDanseMacabre** [comment]
!review

**Review by fernerr — COMMENTED** [review]
I heavily disagree with changing the blaster sprites to use an ugly blue furnishing that will blend in with the armor of the tcfl, and be a general downgrade. Guns need to be kept visible from their user, not everything needs to be turned boring and tacticool.

As one of the only factions that uses wood furnishings for their guns, it adds to their uniqueness and it fits with the theme of being a low budget militia using outdated weaponry in the first place. Plus, the legion are not the only users of blaster weaponry.

If you disagree with this assessment, you need to put up a forum thread and link it here, as this it not the place for that kind of back and forth.

**Review by fernerr — CHANGES_REQUESTED** [review]
I know it's a pain in the ass, but if you intend to change the names of ammunition like this, the paths themselves need to be changed too, for example:

 /obj/item/ammo_magazine/**c45** -> /obj/item/ammo_magazine/**c40**

And so on, for the sake of consistency and making it easy to find them in code.
If you run into map errors resulting from this, do let me know and I'll help out if needed.

**Comment by TheDanseMacabre** [comment]
https://forums.aurorastation.org/topic/14814-feedback-thread-danses-gun-changes/

File: code/modules/projectiles/ammunition/boxes.dm [hunk 28]
```diff
@@ -34,177 +34,177 @@
...
-/obj/item/ammo_magazine/c45m
-	name = "magazine (.45)"
+/obj/item/ammo_magazine/c40m
+	name = "magazine (.40)"
 	icon_state = "45"
 	mag_type = MAGAZINE
-	ammo_type = /obj/item/ammo_casing/c45
+	ammo_type = /obj/item/ammo_casing/c40
 	matter = list(DEFAULT_WALL_MATERIAL = 525) //metal costs are very roughly based around 1 .45 casing = 75 metal
 	caliber = ".45"
```
Comment (Geevies) [inline code/modules/projectiles/ammunition/boxes.dm:63]:
```suggestion
	caliber = ".40"
```

File: code/modules/projectiles/ammunition/boxes.dm [hunk 29]
```diff
@@ -34,177 +34,177 @@
...
-	max_ammo = 7
+	max_ammo = 10
```
Comment (Geevies) [inline code/modules/projectiles/ammunition/boxes.dm:64]:
```suggestion
	max_ammo = 7
```

Comment (Geevies) [reply code/modules/projectiles/ammunition/boxes.dm:64]: these already shred antags enough as-is

File: code/modules/projectiles/ammunition/boxes.dm (hunk 28 above)
Comment (Geevies) [reply code/modules/projectiles/ammunition/boxes.dm:63]: a bunch of these calibers didn't change, despite the names doing

**Review by Geevies — CHANGES_REQUESTED** [review]

**Pull request closed without merging**